code/modules/xenoarcheaology/tools/tools.dm

```diff
@@ -310,3 +310,41 @@
 		usr << browse(null, "window=locater")
 
 	updateSelfDialog()
+
+/obj/item/device/xenoarch_multi_tool
+	name = "xenoarcheology multitool"
+	desc = "Has the features of the Alden-Saraspova counter, a measuring tape, and a depth analysis scanner all in one!"
+	icon_state = "ano_scanner2"
+	item_state = "lampgreen"
+	icon = 'icons/obj/xenoarchaeology.dmi'
+	origin_tech = list(TECH_MAGNET = 3, TECH_ENGINEERING = 3, TECH_BLUESPACE = 2)
+	matter = list(DEFAULT_WALL_MATERIAL = 10000,"glass" = 5000)
+	w_class = ITEMSIZE_SMALL
+	slot_flags = SLOT_BELT
+	var/scanning = 1
+	var/measuring = 0
```

Comment:
If `measuring = !scanning` is a tautology (Always true except in cases where it's bugged), i.e., if `(measuring ^ scanning) = 1` (Exclusive OR), then you only need one variable to capture the mode. 

(And can anyways use defines to store an integer state in that variable), as seen below
(You can use whatever string you feel like, following the usual rules of making defines, these are just examples)
```#define XA_MULTI_SCANNING 0
#define XA_MULTI_MEASURING 1```

(In the declaration)
`var/mode = XA_MULTI_SCANNING`

(In use)
```if(mode == XA_MULTI_SCANNING)
[Handle scanning]
else if(mode == XA_MULTI_MEASURING)
[Handle measuring]```

Suppose I should note that for this particular instance, #defines aren't strictly necessary, but I would prefer only one variable, since it has a binary state (Scanning or measuring)